packages/admin-ui/src/app/_services_and_types/participants.service.ts

```diff
@@ -43,6 +48,73 @@ export class ParticipantsService {
 		// this._http.
 	}
 
+	validateSettlementInitiationFile(
+		formData: FormData
```

Comment:
Services must receive properly typed structures, FormData is for the webpage components.